src/app/modules/rup/services/elementosRUP.service.ts

```diff
@@ -206,5 +206,16 @@ export class ElementosRUPService {
         }
     }
 
+    /** TODO validar concepto para solicitud de derivación
+     * concepto temporal: 44383000 - 'pedido de interconsulta'
+     */
+    getConceptoDerivacion() {
+        return {
+            conceptId: '44383000',
+            term: 'derivación del paciente para interconsulta',
+            fsn: 'derivación del paciente para interconsulta (procedimiento)',
+            semanticTag: 'procedimiento'
```

Comment:
Que les parece si usamos esta concepto? @plammel  @liquid36 @silviroa  @kroce  @condorpiedra 

   traslado del paciente (procedimiento)
SCTID: 107724000

107724000 | traslado del paciente (procedimiento) |
  es   traslado del paciente (procedimiento)
  es   traslado del paciente
  es   Cumplimiento de los pasos necesarios para trasladar a un paciente entre establecimientos asitenciales
  en   Transfer of patient
  en   Transfer of patient procedure
  en   Patient transfer
  en   Performance of the steps necessary to transfer a patient between locations of care delivery
  en   Patient transfer (procedure)
  en   Transfer care